components/automate-ui/src/app/pages/integrations/aws-form/aws-form.component.html

```diff
@@ -13,7 +13,7 @@
     <input type="checkbox" formControlName="no_creds">
   </label>
 
-  <ng-container *ngIf="showInstanceCreds()" [formGroup]="formGroup.controls.credentials">
+  <ng-container *ngIf="showInstanceCreds()" [formGroup]="credentialsFormGroup()">
```

Comment:
The common issue was that the RHS is an AbstractControl and that is not compatible with the LHS, which needs a FormGroup. As far as I could tell, there is no way to do a cast within the confines of a template expression (did not like either `<FormGroup>` prefix or `as FormGroup` suffix), so had to do it in the code-behind.

Reply:
I see your note here - and it relates to my comment below.  I'm curious if there may be a way to limit the function call loops